Gordon360/Services/ActivityService.cs

```diff
@@ -25,12 +27,16 @@ public class ActivityService : IActivityService
         private readonly CCTContext _context;
         private readonly IConfiguration _config;
         private readonly IWebHostEnvironment _webHostEnvironment;
+        private readonly IServer _server;
+        private readonly ServerUtils _serverUtils;
 
-        public ActivityService(CCTContext context, IConfiguration config, IWebHostEnvironment webHostEnvironment)
+        public ActivityService(CCTContext context, IConfiguration config, IWebHostEnvironment webHostEnvironment, IServer server, ServerUtils serverUtils)
         {
             _context = context;
             _config = config;
             _webHostEnvironment = webHostEnvironment;
+            _server = server;
```

Comment:
```suggestion
        public ActivityService(CCTContext context, IConfiguration config, IWebHostEnvironment webHostEnvironment, ServerUtils serverUtils)
        {
            _context = context;
            _config = config;
            _webHostEnvironment = webHostEnvironment;
```
I don't think you need the IServer anymore, right?